src/pages/NodeChecker/Client.tsx

```diff
@@ -0,0 +1,58 @@
+import {EvaluationSummary, NodeCheckerClient} from "aptos-node-checker-client";
+import {GlobalState} from "../../GlobalState";
+
+export const DEFAULT_NHC_INSTANCE =
```

Comment:
What happens when someone switches networks?

Reply:
If they select "local" it tries to use a local NHC. If they select anything else, it uses the prod NHC (this default instance value).